.pre-commit-config.yaml

```diff
@@ -9,3 +9,8 @@ repos:
       - id: detect-private-key
       - id: trailing-whitespace
       - id: end-of-file-fixer
+  - repo: https://github.com/brokenpip3/pre-commit-hooks
```

Comment:
What is the purpose of this?

Reply:
it's a pre-commit hook I created to avoid commit github action without pinning the hash: https://github.com/brokenpip3/pre-commit-hooks?tab=readme-ov-file#github-actions-hash 
anyway I removed it